Name: Hide Amazon MP3
Version: 1.4.0-1
Author: cryptk
Description: This patch will hide the Amazon MP3 application.

--- .orig/usr/palm/applications/com.palm.app.amazonstore/appinfo.json
+++ /usr/palm/applications/com.palm.app.amazonstore/appinfo.json
@@ -4,6 +4,7 @@
 	"main": "index.html",
 	"id": "com.palm.app.amazonstore",
 	"icon": "icon.png",
-	"splashicon": "icon-256x256.png"
+	"splashicon": "icon-256x256.png",
+	"visible": false
 }
 
